test/unit/sql/where.test.js

```diff
@@ -908,6 +908,24 @@ suite(Support.getTestDialectTeaser('SQL'), () => {
       });
     }
 
+    if (current.dialect.supports.REGEXP) {
+      suite('$regexp', () => {
+        testsql('username', {
+          $regexp: '^sw.*r$'
```

Comment:
Ok, test with \n has passed (other than the unrelated test). Is there any other specific literal you want tested?